github-runner-manager/src/github_runner_manager/platform/__init__.py

```diff
@@ -2,3 +2,18 @@
 #  See LICENSE file for licensing details.
 
 """Package for platform providers."""
+from enum import Enum
+
+
+class Platform(str, Enum):
+    """Enum for supported platforms.
+
+    Attributes:
+        GITHUB: GitHub platform.
+        JOBMANAGER: JobManager platform.
+        MULTIPLEXER: Multiplexer platform that uses several providers simultaneously.
+    """
+
+    GITHUB = "github"
+    JOBMANAGER = "jobmanager"
+    MULTIPLEXER = "multiplexer"
```

Comment:
I think `MULTIPLEXER` belongs rather to the `Mode` rather than the `Platform` enum class.

Perhaps we can create another dataclass for which `Mode` the application is running in, or, we make the "Job" of application be multiplexed by default in the future, and define what platform the job should be spawned for.

Reply:
Also after discussion with @javierdelapuente , I've moved this enum to `multiplexer_provider.py`, as it makes only sense in the context of the multiplexer. Does this solve your concern, too?